### PR TITLE
16 add further descriptions about model parameters in documentation

### PR DESCRIPTION
**Pull request by gracezhihuizhao**
[Available Curves in tcplfit2.pdf](https://github.com/USEPA/CompTox-ToxCast-tcplFit2/files/12652787/Available.Curves.in.tcplfit2.pdf)

After discussion with SME's, creating a vignette that contains the parameter descriptions and plots that shows how curve shape changes as parameter values change. Included rendered pdf for easy review.

### CODE REVIEW

File: vignettes/curve-shapes.Rmd [hunk 1]
```diff
@@ -0,0 +1,477 @@
...
+    geom_line(aes(y = y1, color = "p=0.87")) +
+     geom_line(aes(y = y6, color = "p=0.9")) +
+   geom_line(aes(y = y7, color = "p=0.7")) +
+  labs(x = "x", y = "y") +
+  theme(legend.position = c(0.1,0.8)) +
+  scale_color_manual(name='p values',
+                     breaks=c('p=0.87', 'p=0.9', 'p=0.7'),
+                     values=c('p=0.87'='black', 'p=0.9'='red', 'p=0.7'='green'))
+
+#exp3_plot1
```
Comment (sedavid01) [inline vignettes/curve-shapes.Rmd:139]: Question... Is there a reason that these plot objects are commented out?

File: vignettes/curve-shapes.Rmd
Comment (sedavid01) [inline vignettes/curve-shapes.Rmd:1]: Use grid to show related figures side-by-side. Make sure to add captions to figures as well.

Comment (sedavid01) [inline vignettes/curve-shapes.Rmd:1]: Suggest adding brief comments to the code chunks.

**Review by sedavid01 — CHANGES_REQUESTED** [review]
Provided some comments to get you started on polishing this vignette.  Remember that you are developing documentation to help users of many levels from brand new users and newcomers to dose-response modeling, to 'wet-lab' individuals, and experienced scientists/modelers.  Provide enough information to explain what is going on but concise enough that it is not burdensome.

This is a good start, just continue building and polishing the vignette you have here.

**Review by sedavid01 — CHANGES_REQUESTED** [review]
The updates are looking good so far.  Especially the section linking.  However, there is still quite a bit of polishing that can be done to improve the quality of the vignette.  Please review the suggestions and adjust accordingly.

Please reach out with any questions or clarifications that are necessary.

File: vignettes/curve-shapes.Rmd
Comment (sedavid01) [inline vignettes/curve-shapes.Rmd:1]: Suggest making sure the legend matches the order (from top to bottom) the color of the respective curve shapes. See "a" plot for Figure 6.

**Review by sedavid01 — CHANGES_REQUESTED** [review]
Great improvements!  Please continue polishing up - there are some pointed suggestion in the comments here.

**Comment by gracezhihuizhao** [comment]
> Great improvements! Please continue polishing up - there are some pointed suggestion in the comments here.
@sedavid01 I made some changes to the plots and fixed some typos.. it's ready for review

**Review by sedavid01 — APPROVED** [review]
Made some minor changes, but overall looks good from my end.  Approving.
Please wait to merge until there is input from SME's that use the package.

**Comment by sedavid01** [comment]
This PR has been approved and outstanding for a while.  I assume SME's have no further comments at this time.  Thus, merging into "dev".  Further refinements may be requested and  scoped in later issues.

**Pull request merged**